ui/src/app/verify/result/page.tsx

```diff
@@ -4,33 +4,32 @@ import { Loader } from '@components'
 import { useStore, useVerifyProof } from '@hooks'
 
 export default function Page() {
-  const { proof, publicSignals } = useStore()
-  const { error, isError, isLoading, valid } = useVerifyProof()
+  const { proof } = useStore()
+  const { isValid } = useVerifyProof()
 
-  if (proof === null || publicSignals === null)
+  if (proof === null)
     return (
       <div>
-        Missing proof and public signals JSON files, go back to{' '}
-        <Link href='/verify' className='is-primary'>
+        Missing proof JSON file, go back to{' '}
+        <Link href='/verify' className='nes-text is-primary'>
```

Comment:
We don't use nes.css anymore. Only tailwind and custom written css rules.
```suggestion
        <Link href='/verify' className='underline'>
```